contracts/cw721-sellable/src/lib.rs

```diff
@@ -91,6 +93,8 @@ mod entry {
             .contract_metadata
             .save(heap_deps.storage, &msg.contract_metadata)?;
```

Comment:
Why did we change the name of `deps_mut` to `heap_deps`? It doesn't actually save to a heap, it is stored to hard disk/cache.